index-providers/modeshape-elasticsearch-index-provider/src/main/java/org/modeshape/jcr/index/elasticsearch/TransportAddress.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * ModeShape (http://www.modeshape.org)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.modeshape.jcr.index.elasticsearch;
+
+import java.net.InetAddress;
+import java.net.UnknownHostException;
+import org.elasticsearch.common.transport.InetSocketTransportAddress;
+
+/**
+ *
+ * @author kulikov
+ */
+public class TransportAddress {
```

Comment:
u'r right, I didn't notice that constant is just the default value
still, I'm not 100% sure a separate class is required for testing essentially a `string.split` method